utils.ts

```diff
@@ -0,0 +1,172 @@
+import fetch from 'node-fetch'
+import { getDistance } from 'geolib'
+import { fromCoordinates } from '@conveyal/lonlat'
+import type { LonLatOutput } from '@conveyal/lonlat'
+import type { Feature, FeatureCollection, Position } from 'geojson'
+
+// Types
+export type ServerlessEvent = {
+  headers: Record<string, string>
+  queryStringParameters: Record<string, string>
+}
+export type ServerlessCallbackFunction = (
+  error: string | null,
+  response: {
+    body: string
+    headers: Record<string, string>
+    statusCode: number
+  } | null
+) => void
+export type ServerlessResponse = {
+  body: string
+  headers: Record<string, string>
+  statusCode: number
+}
+
+/**
+ * Executes a request on a Pelias instance
+ * @param baseUrl URL of the Pelias instance, without a trailing slash
+ * @param service The endpoint to make the query to (generally search or autocomplete)
+ * @param query   The rest of the Pelias query (any GET paremeters)
+ * @returns       Pelias response decoded from JSON
+ */
+export const fetchPelias = async (
+  baseUrl: string,
+  service: string,
+  query: string
+): Promise<FeatureCollection> => {
+  try {
+    const response = await fetch(`${baseUrl}/${service}?${query}`, {})
+    return await response.json()
+  } catch {
+    console.warn(`${baseUrl} failed to return response`)
+    return { features: [], type: 'FeatureCollection' }
+  }
```

Comment:
This is going to end up swallowing errors that we may want to know about. Maybe we should add some Bugsnag reporting here.

Reply:
Added in 79923da2c255284e580df7907b9ec45b107762af